libs/statistics/include/hpx/statistics/rolling_max.hpp

```diff
@@ -0,0 +1,116 @@
+//  Copyright (c) 2017 Hartmut Kaiser
```

Comment:
Is the original file still in `hpx/util/rolling_max.hpp` (asking because the diff doesn't show it as being moved)? Same for `rolling_min.hpp`.

Reply:
Yep I did only `mv` commands